ethereum/gateway-eth-ts/src/GatewayTs.ts

```diff
@@ -14,7 +16,8 @@ export class GatewayTs extends GatewayTsBase {
   ) {
     super(provider, wallet, options);
 
-    super.setGasLimit();
+    // eslint-disable-next-line no-void
```

Comment:
disable lint and add void? Would have only expected one.

Reply:
what does that void actually do?

Reply:
I originally added void with no disable, but then eslint threw an error for having void in there. I suppose I could've just disabled it without the void...

Reply:
ok why did you add void in the first place? What does it do? I only understand it in regard to a function definition.